src/avram/params.cr

```diff
@@ -38,4 +38,8 @@ class Avram::Params
   def nested_file(key) : Hash(String, String)
     @hash
   end
+
+  def has_key_for?(operation : Avram::Operation.class | Avram::SaveOperation.class) : Bool
+    nested?(operation.param_key).any?
```

Comment:
Seems like this implementation can go in `Avram::Paramable`?

Reply:
I thought about that too. I actually wrote the code in Lucky first, then came and did this. I think I'll do that.